src/components/Button/Button.tsx

```diff
@@ -198,8 +207,29 @@ export function Button({
   const { Link } = useComponents();
   const LinkButton = BaseButton.withComponent(Link);
   const ButtonElement = props.href ? LinkButton : BaseButton;
+
+  /** Set default value for tracking dispatch */
+  const { label = undefined, component = 'button', data = undefined } =
```

Comment:
I couldn't find in the docs where `label` can be undefined for click events. And looks like `data` was renamed to `customParameters`

Reply:
Here we need to think how often people will use only the label vs label + additional fields. if we see just label as the most common use case, I'd move it to a prop such as `tracking-label`.

Reply:
Can the component be overwritten?

Reply:
> I couldn't find in the docs where `label` can be undefined for click events. And looks like `data` was renamed to `customParameters`

I set `label` to undefined to follow this rule: To ensure that previously set parameters are reset and won’t be persisted in the GTM, all parameters must be passed as undefined (JavaScript literal) with every push, if no value is defined.

Reply:
> Here we need to think how often people will use only the label vs label + additional fields. if we see just label as the most common use case, I'd move it to a prop such as `tracking-label`.

Ahhhh got it. Let me try it this way. Thanks!!

Reply:
> Can the component be overwritten?

It could be overwritten like this: https://github.com/sumup-oss/circuit-ui/pull/598/files#diff-1524ebf97b749e7e1d8e1a48621c0befR148

Is this what you refer to??

Reply:
I disagree with you @fernandofleury, I'd prefer to keep the tracking data in one prop.

```ts
<Button trackingPayload={{ label: 'foo' }} />
// vs
<Button trackingLabel="foo" trackingPayload={{ bar: 'foo' }} />
```

How about we rename `trackingPayload` to `tracking`, to save some chars?

Reply:
rename trackingPayload to tracking make sense to me. thanks!!

Reply:
@lucent1090 just asked over #tracking, and in their words:

>I'd argue against it. This is one of the current factors why the data quality is poor at times.

Reply:
So we won't have clicks without labels

Reply:
Thank you for this. I'll fix it today.